improver/__init__.py

```diff
@@ -76,9 +77,9 @@ def __call__(self, *args, **kwargs):
                 Positional arguments.
             **kwargs:
                 Keyword arguments.
+
```

Comment:
The issue here is that Cube is yet to be defined so it can't be annotated. The ability to annotate without evaluation was going to be available in Python 3.10 with the adoption of PEP 563 but this has been postponed to Python 3.11.  It is also available with

```python
from __future__ import annotations
```

but this only works for Python 3.7+.

Reply:
Okay, thanks of explaining.

Reply:
No worries